mixin-utils/utils.libsonnet

```diff
@@ -411,20 +409,17 @@ local g = import 'grafana-builder/grafana.libsonnet';
       if 'alert' in rule && std.objectHas(overrides, rule.alert)
       then rule + overrides[rule.alert]
       else rule,
-    local overrideInGroup(group) = group { rules: std.map(overrideRule, super.rules) },
+    local overrideInGroup(group) = group + { rules: std.map(overrideRule, super.rules) },
     prometheusAlerts+:: {
       groups: std.map(overrideInGroup, super.groups),
     },
   },
 
   removeAlerts(alerts):: {
-    local removeRule(rule) =
-      if 'alert' in rule && std.objectHas(alerts, rule.alert)
```

Comment:
Oh, I'm breaking something here, this function takes an object as input, not an array...

Reply:
covered, it is not backwards compatible.